### PR TITLE
LPS-74457 Enable language keys check for subrepositories

### PR DESCRIPTION
**Pull request by petershin**
Hi Hugo,

The faro team would prefer enabling the language keys check for subrepos.

/cc @shinnlok

### CODE REVIEW

**Comment by petershin** [comment]
ci:reopen

**Comment by hhuijser** [comment]
@petershin @shinnlok, what about we make this configurable?

With this change, we now have situations where locally SF gives different results than on Jenkins, because locally SF cannot access language.properties in portal-impl, which some engineers find annoying.

**Comment by petershin** [comment]
@hhuijser - Does this [commit](https://github.com/hhuijser/liferay-portal/pull/3039/commits/94452713be983774913e0fedcc2f5d7915f07527) ensure that it works the same way everywhere?

**Comment by hhuijser** [comment]
@petershin, I don't think so.
I believe that when Jenkins tests, it does not 'see' is as subrepo since the code is dropped inside portal.

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/checks/LanguageKeysCheck.java [hunk 1]
```diff
@@ -56,11 +56,7 @@ protected String doProcess(
 			String fileName, String absolutePath, String content)
 		throws Exception {
 
-		if (!isSubrepository() &&
-			!absolutePath.contains("/modules/private/apps/")) {
```
Comment (shinnlok) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/checks/LanguageKeysCheck.java]: @hhuijser wouldn’t the solution be just to take this line as well and move it to the check below?

**Comment by petershin** [comment]
Added the check @shinnlok mentioned for Jenkins

**Comment by hhuijser** [comment]
I'll look into this a bit more after the weekend

**Comment by hhuijser** [comment]
Sent to Brian: https://github.com/brianchandotcom/liferay-portal/pull/52402

**Pull request closed without merging**